src/scripts/main.js

```diff
@@ -1,3 +1,203 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('table');
+const headers = [...table.tHead.rows[0].cells];
+// const thead = document.querySelector('thead');
+const tbody = table.tBodies[0];
+const tableRows = [...tbody.rows];
+
+// Implement table sorting by clicking on the title (in two directions)
+
+function sortTable(columnIndex, sortOrder) {
+  tableRows.sort((rowA, rowB) => {
+    const cellA = rowA.querySelectorAll('td')[columnIndex].textContent;
+    const cellB = rowB.querySelectorAll('td')[columnIndex].textContent;
+
+    if (sortOrder === 'asc') {
+      return cellA.localeCompare(cellB, undefined, { numeric: true });
+    } else {
+      return cellB.localeCompare(cellA, undefined, { numeric: true });
+    }
+  });
+
+  tableRows.forEach((row) => {
```

Comment:
Instead of removing and appending rows one by one, you can directly set tbody.innerHTML to an empty string before appending sorted rows.